src/ApiCompat/Rules/Compat/CannotMakeMoreVisible.cs

```diff
@@ -47,7 +47,7 @@ public override DifferenceType Diff(IDifferences differences, ITypeDefinition im
             // If implementation is protected then contract must be protected as well. 
             if (impl.GetVisibility() == TypeMemberVisibility.Family)
             {
-                if (contract.GetVisibility() != TypeMemberVisibility.Family)
+                if (contract.GetVisibility() != TypeMemberVisibility.Family && contract.GetVisibility() != TypeMemberVisibility.FamilyOrAssembly)
```

Comment:
I don't think types can really be `FamilyOrAssembly` (unless we go through this code path for nested types). At any rate it probably doesn't hurt anything to include it here. I would update the condition for the outter if as well like I called in the other comment.